packages/clerk-js/src/core/modules/checkout/__tests__/manager.spec.ts

```
@@ -0,0 +1,634 @@
+      expect(state.fetchStatus).toBe('fetching');
+
+      // Try to clear while operation is pending
+      manager.clearCheckout();
+
+      // State should not be cleared
+      state = manager.getCacheState();
+      expect(state.isStarting).toBe(true);
+      expect(state.fetchStatus).toBe('fetching');
+
+      // Resolve the operation
+      resolveOperation!(mockCheckout);
```

ESLint

[error] Forbidden non-null assertion.

(@typescript-eslint/no-non-null-assertion)